src/components/Alteration/TransactionalFolioNumber.vue

```diff
@@ -88,7 +88,7 @@ export default class TransactionalFolioNumber extends Mixins(CommonMixin) {
   private async onFolioNumberChanged (val: string): Promise<void> {
     this.setTransactionalFolioNumber(val)
     // wait for form field to update itself before checking validity
-    await Vue.nextTick()
+    await this.$nextTick()
```

Comment:
I made this change in all files (except tests) to reduce the number of imports. (Vue is actually inherited by the mixins that this class components extends.)